spec.html

```diff
@@ -319,6 +319,12 @@ <h3>Non-terminals</h3>
           <td class="fix ex">unsigned_byte(8)</td>
           <td>Sensitive</td>
         </tr>
+        <tr>
+          <td class="fix nt"></td>
+          <td class="fix op">|</td>
+          <td class="fix ex">unsigned_byte(9)</td>
+          <td><a href="https://www.mongodb.com/docs/manual/reference/bson-types/#binary-data">Vector</a></td>
```

Comment:
S: when the server manual is updated there will be an entry for 9
see https://github.com/10gen/docs-mongodb-internal/pull/9411

Reply:
this was just merged :)

Reply:
I'll wait to merge this PR until the changes are deployed on the Server docs site.

Reply:
Looks like it's published: https://www.mongodb.com/docs/manual/reference/bson-types/#binary-data

Reply:
It would be nice to have at least __some__ description given to Vector. Perhaps:

Vector - densely packed arrays of numbers, all of the same type

Reply:
@caseyclements it appears that you are commenting on an outdated snippet from this PR - please look at the files changed section to see my note about what vector is

Reply:
I am not responsible for editing the Server manual docs, but can cc @kanchana-mongodb here

Reply:
Thanks @rustagir . I will update the server docs.